snappy/install.go

```diff
@@ -119,7 +119,9 @@ func doInstall(name, channel string, flags LegacyInstallFlags, meter progress.Me
 		return "", err
 	}
 
-	snap, err := mStore.Snap(name, channel, nil)
+	// devmode false preserves the old behaviour but we might want
+	// it to be set from flags instead.
+	snap, err := mStore.Snap(name, channel, false, nil)
```

Comment:
what stopped me is that this is the only method that'd take the struct.

About the flags, that'd be hard because the flags are defined over in snapstate, which imports store